java/arcs/core/data/testdata/Manifest2ProtoTest.arcs

```diff
@@ -0,0 +1,50 @@
+// This files is a part of 3 files suite of testdata:
+// - Manifest2ProtoTest.arcs (this file)
+// - Manifest2ProtoTest.pb.bin
+// - Manifest2ProtoTest.textproto
+//
+// On the TypeScript side we serialize .arcs file and validate it equals the .pb.bin file.
```

Comment:
It seems like this will make failing diffs very hard to read. Is there a good method for updating all these files?

Reply:
As you noted provided in the test :)